site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        sling:configPropertyInherit="{Boolean}false"
+        prefixPath="/content/aem-site-template-standard.theme/_default">
+        <items
+            jcr:primaryType="nt:unstructured"
+            sling:configCollectionInherit="{Boolean}true"
+            sling:resourceType="nt:unstructured">
```

Comment:
Is the `sling:resourceType` necessary? Will it not default to the primary type anyway?

Reply:
removed in f9e8472ad64dedf84184babaeeb196e9f073df26. Was added by the CA Config Editor